pages/docs/manual/latest/exception.mdx

```diff
@@ -62,20 +65,32 @@ You can directly match on exceptions _while_ getting another return value from a
 <CodeTab labels={["ReScript", "JS Output"]}>
 
 ```res
```

Comment:
should this be `prelude` ?

Reply:
@kamweti reopening the question

Reply:
@cristianoc sorry i thought i posted the comment earlier

Reply:
`prelude` is exactly what makes sure future changes don't introduce compilation errors on the code
That's why I suggested to add it

Reply:
ok sure, i'm gonna add it